assets/xml/objects/object_stk.xml

```diff
@@ -1,110 +1,138 @@
+        <Animation Name="gSkullKidCallDownMoonStartAnim" Offset="0x2774" /> <!-- Original name is "sk_moon_hikiyose" ("draw or pull the moon") -->
+        <Animation Name="gSkullKidCallDownMoonLoopAnim" Offset="0x3068" /> <!-- Original name is "sk_moon_hikiyose_loop" -->
+
+        <!-- DisplayLists for an unused Link Mask -->
+        <DList Name="gSkullKidLinkMask1DL" Offset="0x39A0" />
+        <DList Name="gSkullKidLinkMask2DL" Offset="0x46B0" />
+        <DList Name="gSkullKidLinkMask3DL" Offset="0x53C0" />
+
+        <!-- Majora's Mask Eyes DisplayList and Texture -->
+        <DList Name="gSkullKidMajorasMaskEyesDL" Offset="0x5870" />
+        <Texture Name="gSkullKidMajorasMaskEyeTex" OutName="skull_kid_majoras_mask_eye" Format="rgba16" Width="32" Height="32" Offset="0x5920" />
+
+        <!-- DisplayLists for Majora's Mask -->
+        <DList Name="gSkullKidMajorasMask1DL" Offset="0x6BB0" />
+        <DList Name="gSkullKidMajorasMask2DL" Offset="0x79F0" />
+
+        <!-- Unused Eyes DisplayList and Texture -->
+        <DList Name="gSkullKidUnusedEyesDL" Offset="0x7E10" />
+        <Texture Name="gSkullKidUnusedEyeTex" OutName="skull_kid_unused_eye" Format="rgba16" Width="8" Height="8" Offset="0x7EA0" />
+
+        <!-- Various Hand DisplayLists -->
+        <DList Name="gSkullKidUntexturedLeftHand" Offset="0x84C0" />
+        <DList Name="gSkullKidOpenLeftHand" Offset="0x87B0" />
+        <DList Name="gSkullKidFluteHoldingLeftHand" Offset="0x8A80" />
+        <DList Name="gSkullKidTwoFingersExtendedLeftHand" Offset="0x90C0" />
+        <DList Name="gSkullKidMaskHoldingRightHand" Offset="0x9710" />
+        <DList Name="gSkullKidUntexturedRightHand" Offset="0x9AC0" />
+        <DList Name="gSkullKidOcarinaHoldingRightHand" Offset="0x9DA0" />
+
+        <!-- Head DisplayLists with two different expressions -->
+        <DList Name="gSkullKidNormalEyesDL" Offset="0xA530" />
+        <DList Name="gSkullKidNormalHeadDL" Offset="0xA5C0" />
+        <DList Name="gSkullKidLaughingEyesDL" Offset="0xAE30" />
+        <DList Name="gSkullKidLaughingHeadDL" Offset="0xAEC0" />
+
+        <Animation Name="gSkullKidTelescopeLookUpStartAnim" Offset="0xBB2C" /> <!-- Original name is "stk_miageru" ("look up") -->
+        <Animation Name="gSkullKidTelescopeLookUpLoopAnim" Offset="0xC964" /> <!-- Original name is "stk_miageru_loop" -->
+        <DList Name="gSkullKidOcarinaOfTimeDL" Offset="0xCAD0" />
+        <Animation Name="gSkullKidTPoseAnim" Offset="0xCC94" /> <!-- Original name is "stk_pause" -->
+
+        <!-- Skull Kid Limb DisplayLists -->
+        <DList Name="gSkullKidTorsoDL" Offset="0xF6F0" />
+        <DList Name="gSkullKidNeckDL" Offset="0xF8B8" /> <!-- Renders a single triangle instead of a normal-looking limb -->
+        <DList Name="gSkullKidOoTStyleHeadDL" Offset="0xF908" /> <!-- Unused in the final game -->
```

Comment:
No, it's not the same, but it's similar. I'll admit this is probably the weakest name in all of these objects. Let me show what I mean with visuals:

OoT:
![image](https://user-images.githubusercontent.com/12245827/172739270-d1ab2530-7c83-4d36-a4fe-b334429d976f.png)

gSkullKidOoTStyleHead:
![image](https://user-images.githubusercontent.com/12245827/172739358-8abe2cf3-f730-46e5-8524-bf0d49fb644e.png)

The actual head that gets used in-game:
![image](https://user-images.githubusercontent.com/12245827/172739428-ec45b12f-a822-43eb-9fe1-ca318931492f.png)

Now, the black color of the first two heads isn't what's important (since in the Japanese version, he still has a black head in-game), but the first two heads both have a orange-yellow-ish(?) mouth painted onto the head, whereas the in-game one has a three-dimensional beak.

Reply:
Ah, I see, so the Japanese version will use the third head but with the black colour. So it's really about beak vs lips?

Reply:
Yes, that's correct. OoT and the second head use lips, and the third (in-game) head uses a beak.

Reply:
I'd say at least comment on that then, if not name based on that.